static/js/click_handlers.js

```diff
@@ -619,6 +619,21 @@ export function initialize() {
     stream_popover.register_click_handlers();
     notifications.register_click_handlers();
 
+    $("body").on(
+        "click",
+        "#next_unread_container .next_topic, #next_unread_container .next_pm",
+        (e) => {
+            if ($(e.target).hasClass("next_topic")) {
+                narrow.narrow_to_next_topic();
+            } else {
+                narrow.narrow_to_next_pm_string();
+            }
+
+            e.stopPropagation();
+            e.preventDefault();
+        },
+    );
```

Comment:
This might be more readable with two similar click handlers rather than doing the `e.target` check.

Reply:
Done.